src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -443,6 +465,57 @@ private static String getNodeClassNameWithoutContext(ParseTree node) {
     }
 
     /**
+     * Method to search for a missed HTML tag which could possibly be responsibe for ANTLR error.
```

Comment:
`could possibly be responsibe` - I don't like this wording, as it sounds too uncertain.